pkg/studyengine/expressions.go

```diff
@@ -1316,7 +1316,7 @@ func (ctx EvalContext) generateRandomNumber(exp types.Expression) (val float64,
 	}
 	max := int(arg2.(float64))
 
-	rand.Seed(time.Now().UnixNano())
+	rand.Seed(Now().UnixNano())
```

Comment:
Is this also intended here? Maybe could be useful to be able set a seed directly for debugging, otherwise the time here is just used for improving randomness.

Reply:
I agree, this is out of scope of time management. I will revert this point, and maybe it should be done separetly because it's deprecated since go 1.20, seed is now set to a random value at startup (not 1 as before 1.20)